### PR TITLE
Fix plist for 64-bit Mac builds to fix auto-launch behavior

### PR DESCRIPTION
**Pull request by kswenson**
Fix plist for 64-bit Mac builds to fix auto-launch behavior [#160083859]

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -212,7 +219,6 @@
                mainclassname="${APP_JAVA_CLASS}">
 
       <arch name="x86_64" />
-      <arch name="i386" />
```
Comment (mklewandowski) [inline build.xml]: I imagine "i386" is mostly obsolete at this point?

File: resources/Info-merge.plist [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mklewandowski) [inline resources/Info-merge.plist]: These were simply additional, required plist entries that were needed for the 64 bit version to run properly?

Comment (kswenson) [reply resources/Info-merge.plist]: For the 32-bit build, we just copy the `Info.plist` we have in `resources` to the executable. For the 64-bit build, `AppBundler` generates an `Info.plist` that doesn't contain everything we need, so we add the items we need that aren't generated by AppBundler, which is a subset of what's in our original `Info.plist`.

File: build.xml (hunk 1 above)
Comment (kswenson) [reply build.xml]: We're not trying to support a universal binary -- there are too many differences between the 32/64-bit builds, including JDK version -- so 64-bit support is all we need.

Comment (mklewandowski) [reply build.xml]: Got it.  So all 32 bit references are presumably obsolete for the Mac build as 32 bit builds are no longer supported.

File: resources/Info-merge.plist (hunk 2 above)
Comment (mklewandowski) [reply resources/Info-merge.plist]: Got it.  Makes sense.

File: build.xml (hunk 1 above)
Comment (mklewandowski) [reply build.xml]: Right, of course.

**Pull request merged**